mathesar_ui/src/component-library/cancel-or-proceed-button-pair/CancelOrProceedButtonPair.scss

```diff
@@ -1,4 +1,5 @@
 .cancel-or-proceed-button-pair {
   display: flex;
-  justify-content: space-between;
+  justify-content: end;
+  gap: 1rem;
```

Comment:
We need to support browsers people currently use, and Safari desktop browser is in use by atleast 9.6% of users (based on [wikipedia](https://en.wikipedia.org/wiki/Usage_share_of_web_browsers)). 

Do not use `gap` or anything not supported by the last 2 versions of all the browsers. (IE can be ignored, since Edge fills for it).

Reply:
Removed here: bf4f022